wrapper/CSharp/wolfSSL-Example-IOCallbacks/wolfSSL-Example-IOCallbacks.cs

```diff
@@ -135,6 +135,67 @@ public static uint my_psk_server_cb(IntPtr ssl, string identity, IntPtr key, uin
         return (uint)4;
     }
 
+    /// <summary>
+    /// Example of a certificate verify function
+    /// </summary>
+    /// <param name="preverify"></param>
+    /// <param name="store">pointer to a WOLFSSL_X509_STORE_CTX</param>
+    /// <returns>size of key set</returns>
+    public static int my_verify_cb(int preverify, IntPtr store)
+    {
+        if (store == IntPtr.Zero)
+        {
+            Console.WriteLine("store is null");
+        }
+
+        Console.WriteLine("Status of certificate verify = " + preverify);
+
+        /* look at the current cert in store */
+        try
+        {
+
+            X509 x509 = wolfssl.X509_STORE_CTX_get_current_cert(store);

```

Comment:
Can we also add the error code information `x509.error`? That is usually required for determining if an override needs to happen.
`Console.WriteLine("Verify Callback Error = " + x509.error);` and also print the error string.

Reply:
added in X509_STORE_CTX_get_error